opencti-platform/opencti-front/lang/front/fr.json

```diff
@@ -3030,20 +3034,20 @@
   "x_opencti_base_score_gt": "CVSS3-Score supérieur à",
   "x_opencti_base_score_lte": "CVSS3-Score inférieur ou égal à",
   "x_opencti_base_severity": "CVSS3-Severity",
-  "x_opencti_detection": "Détection",
   "X_opencti_detection": "Detection",
+  "x_opencti_detection": "Détection",
   "X_opencti_main_observable_type": "Main observable type",
   "x_opencti_main_observable_type": "Type d'observable principal",
   "x_opencti_negative": "Qualification",
   "x_opencti_organization_type": "Type d'organisation",
-  "x_opencti_reliability": "Fiabilité",
   "X_opencti_reliability": "Reliability",
-  "x_opencti_score": "Score",
+  "x_opencti_reliability": "Fiabilité",
```

Comment:
is it possible to use only one of X_opencti_reliability and x_opencti_reliability ? Same for score and workflow below.

Reply:
and detection